source/isFunctor.ts

```diff
@@ -5,5 +5,4 @@ export interface IsFunctor {
 }
 
 export const isFunctor: IsFunctor = x =>
-  (isArray(x) || isObject(x))
-    && isFunction((x as Record<string, unknown> | Array<unknown>).map)
```

Comment:
This lets me not have to do this `as <the things I just tested for>` casting

Reply:
Cool!

A functor is an object with a `map` method :)